distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column_Description.enso

```diff
@@ -0,0 +1,16 @@
+from Standard.Base import all
+
+import Standard.Table.Data.Type.Value_Type.Value_Type
+
+import project.Data.Column_Constraint.Column_Constraint
+
+## Describes a column structure for `Connection.create_table`.
+type Column_Description
+
+    ## Describes a column structure for `Connection.create_table`.
+
+       Arguments:
+       - name: The name of the column.
+       - value_type: The type of the column.
+       - constraints: Additional SQL constraints for the column.
+    Value (name:Text) (value_type:Value_Type) (constraints : Vector Column_Constraint = [])
```

Comment:
> While not for this PR - I think we should invest time in the conversions.

I agree.

> `Column_Description.from that:Pair` converting a Text / Value_Type to it.

I'm not that sure about this. I guess it depends how the 'auto' conversions that Jaroslav is suggesting will work. My worry is - this may be too 'broad'. Since we do not have 'higher order' types, any `Pair` will be attempted to be converted into `Column_Description`, even if it is a `Pair.new 10 20` - and I'm not sure if it is the right course of action.

In general I agree we should use conversions, but I'm not convinced this is the right place. In the GUI the type makes widgets work and makes it clear what is being constructed. In text, it is not that much more typing and gives us much clearer code and stronger type guarantees. Although indeed things like `Table.new` do play well with the `[["X", [...]], ...]` form.

Reply:
I imagined that if the `Pair` was not a `Pair Text Value_Type` we would return a graceful error message in the `from` method.

Using conversions more is only a thought at moment, but I think something we will look at more in future.